src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -50,9 +48,8 @@ void GY50::update()
 
     if (getAbsolute(drift) > kGyroThreshold)
     {
-        float gyroRate                               = static_cast<float>(drift) * kGyroSensitivity;
-        static constexpr auto kGyroscopeReadingScale = 1000.0F;
-        mAngularDisplacement += gyroRate / (kGyroscopeReadingScale / static_cast<float>(interval));
+        float gyroRate = static_cast<float>(drift) * kGyroSensitivity;
+        mAngularDisplacement += gyroRate / (1000.0f / static_cast<float>(interval));
```

Comment:
:warning: **readability-magic-numbers** :warning:
1000.0f is a magic number; consider replacing it with a named constant